File: maven/spring-boot-2.7.0-study/spring-boot-2.7.0-scheduling/src/main/java/com/litongjava/spring/boot/scheduing/controller/IndexController.java
package com.litongjava.spring.boot.scheduing.controller;

import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping("")
public class IndexController {
  @RequestMapping("")
  public String index() {
    return "index";
  }
}

File: maven/java-ee-spring-boot-2.2.4-study/java-ee-spring-boot-2.2.4-kotlin/src/main/kotlin/com/litongjava/spring/boot/kotlin/hello/controller/IndexController.kt
package com.litongjava.spring.boot.kotlin.hello.controller

/**
 * Created by litonglinux@qq.com on 9/4/2021_10:36 PM
 */

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class IndexController {

  @GetMapping("/")
  fun index(): String {
    return "hello kotlin"
  }
}